lib/Javascript/src/Store.ts

```diff
@@ -80,22 +80,10 @@ class Store {
         serviceRecord.characteristics.set(keyForCharacteristic(characteristic), { uuid: characteristic.uuid, characteristic, descriptors: new Map() });
     }
 
-    private findCharacteristic = (key: CharacteristicKey): BluetoothRemoteGATTCharacteristic | undefined => {
-        for (const deviceRecord of this.#devices.values()) {
-            for (const serviceRecord of deviceRecord.services.values()) {
-                const characteristic = serviceRecord.characteristics.get(key)?.characteristic;
-                if (characteristic) {
-                    return characteristic;
-                }
-            }
-        }
-        return undefined;
-    }
```

Comment:
Whoops, must have been in "just get it working" mode when this was dropped here.

Reply:
Just realized why this may have worked previously - the `instance` used to be based on the object ID which would have been unique per-device, but it was changed to be a deterministic sequenced integer which is no longer unique on it's own.